manifests/p/Princessmortix/Cobalt/1.0.4/Princessmortix.Cobalt.locale.en-US.yaml

```diff
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: Princessmortix.Cobalt
+PackageVersion: 1.0.4
+PackageLocale: en-US
+Publisher: Princess Mortix
+PublisherUrl: https://github.com/princessmortix/cobalt
+PublisherSupportUrl: https://github.com/princessmortix/cobalt/issues
+Author: Princess Mortix
+PackageName: cobalt-cli
+PackageUrl: https://github.com/princessmortix/cobalt
+License: MIT License
+LicenseUrl: https://github.com/princessmortix/cobalt/blob/main/LICENSE
+CopyrightUrl: https://github.com/princessmortix/cobalt/blob/main/LICENSE
+ShortDescription: Unofficial cobalt.tools command line client
+Description: Unofficial cobalt command line client made in go. Small footprint & portable for convenience, just like cobalt.
+Moniker: cobalt
+Tags:
+- downloader, cobalt, cli
+ReleaseNotes: 'v1.0.4: Fix display of JSON messages while using flag -j'
+ReleaseNotesUrl: https://github.com/princessmortix/cobalt/releases/latest
+Icons:
+- IconUrl: https://raw.githubusercontent.com/princessmortix/cobalt/main/winres/icon.png
+  IconFileType: png
+  IconResolution: 256x256

```

Comment:
```suggestion
```

Hi @princessmortix,

This feature depends on the `Agreements` section, which I believe hasn't quite been deployed yet. These lines might be readded after it's available.